setup.py

```diff
@@ -3,29 +3,38 @@
 install_requires=[
     'flask-restful>=0.3.0',
     'flask-sqlalchemy >= 2.3.0',
-    'flask',
+    'sqlalchemy_utils>=0.30',
+    'flask>=1.0.0',
     'sqlalchemy>=1.1',
 ]
 
+extras_require = {
+    "docs": ["sphinx >= 1.4", "sphinx_rtd_theme", "sphinx-autodoc-typehints"],
+    "testing": ["codecov", "pytest", "pytest-cov", "pytest-mock", "pytest-datadir", "tox"],
+}
+
 setup(
     name='flask-restalchemy',
     version='0.12.1',
-    packages=find_packages(exclude=['*.tests']),
+    packages=find_packages('src'),
+    package_dir={"": "src"},
```

Comment:
@igortg here's the relevant options regarding using a `src` directory.

Btw we should probably adopt this standard for all of our OSS projects, including the template.